erdpy_network/tokens.py

```diff
@@ -33,7 +33,7 @@ def __init__(self):
         self.creator: IAddress = Address.zero()
         self.supply: int = 0
         self.decimals: int = 0
-        self.royalties: int = 0
+        self.royalties: float = 0
```

Comment:
Just make sure that the responses returned from API vs. returned from Proxy (for the same token) are ~consistent (if possible).

Reply:
Both the api and the gateway return the royalties as `str` but we are converting it to `int` and divide it by 100 to get the real value which can be float. Also added new assert statement to tests.